examples/demos/example27-cell-actions.js

```diff
@@ -52,7 +53,42 @@ const columns = [
   {
     key: 'county',
     name: 'County',
-    width: 200
+    width: 200,
+    cellClass: 'rdg-cell-action',
+    formatter({ row, value }) {
```

Comment:
If you want we can use the new `cellContentRenderer` API here.
```js
    cellContentRenderer({ rowData }) {
      const value = rowData.county;
      if (rowData.id === 'id_0') {
```

Reply:
We probably do not need `cellContentRenderer` once subrows functionality is removed. We can also delete `CellContent` component and use `formatter` as `cellContentRenderer`. We can then replace `defaultCellContentRenderer` with `defaultFormatter`. Wdyt? 

We can merge this PR after the lerna removal PR

Reply:
I agree.

IMO RDG shouldn't access the value from the row object, so we can have renderers like
```js
({ rowData }) => rowData.nested.value;
({ rowData }) => rowData.x + rowData.y;
````
for example. And it might simplify types as well, maybe.